test/unit/WebsocketServer.test.js

```diff
@@ -113,197 +134,117 @@ describe('WebsocketServer', () => {
             wsMock.emit('connection', mockSocket)
         })
 
-        it('sends a resending message before starting a resend', (done) => {
```

Comment:
Dropped coverage?
- sends a `ResendResponseResending` message before starting a resend
- sends a `ResendResponseResent` message when resend is complete
- sends a `ResendResponseNoResend` message when resend is complete